src/main/haskell/matching-logic/src/Kore/MatchingLogic/Signature/MetaK.hs

```diff
@@ -0,0 +1,122 @@
+{-# LANGUAGE TypeFamilies #-}
+{-|
+Description: A signature for the meta-level Kore terms.
+
+Defines a signature for the meta-level Kore terms, but
+currently does not have space for the added symbols that
+will be introduced by lifting object-level definitions.
+-}
+module Kore.MatchingLogic.Signature.MetaK where
+import           Data.Coerce
+
+import           Kore.MatchingLogic.AST (IsSignature)
+import qualified Kore.MatchingLogic.AST as AST
+
+data MetaSort =
```

Comment:
How about reusing the existing data types?
https://github.com/kframework/kore/blob/master/src/main/haskell/language-kore/src/Data/Kore/AST/Common.hs#L159

Reply:
Defining an IsSignature instance for an existing type would be fine, but this module isn't actually used at all so I'll just remove it. Also, what existing type works for labels? Only MLPatternType looks close, and that includes things like `and` which are not labels.